spec/rails_helper.rb

```diff
@@ -45,6 +46,10 @@
 
   config.infer_spec_type_from_file_location!
 
+  config.define_derived_metadata(file_path: Regexp.new("/spec/datatables/")) do |metadata|
```

Comment:
What is this needed for? Maybe we should add a comment here.

Reply:
@compwron It just saves the effort of adding `type: :datatable` to spec files in `/spec/datatables` so that the `DatatableHelper` will be included by default.